src/server/pfs/server/testing/server_test.go

```diff
@@ -5876,16 +5865,18 @@ func TestPFS(suite *testing.T) {
 		})
 	})
 
-	suite.Run("TestLoad", func(t *testing.T) {
-		t.Parallel()
-		env := testpachd.NewRealEnv(t, tu.NewTestDBConfig(t))
-		spec := &load.CommitsSpec{}
-		require.NoError(t, yaml.UnmarshalStrict([]byte(testLoad), spec))
-		msg := random.SeedRand()
-		c := env.PachClient
-		repo := "test"
-		require.NoError(t, c.CreateRepo(repo))
-		require.NoError(t, load.Commits(c, repo, "master", spec), msg)
+	suite.Run("TestLoad", func(subsuite *testing.T) {
```

Comment:
This shouldn't run in the suite with the behavioral tests.  Can we move this to a separate function `TestLoad(t *testing.T)`